### PR TITLE
Fix/README imports

### PR DESCRIPTION
**Pull request by Teoxoy**
https://github.com/material-components/material-components-web/issues/1049 - Updated READMEs to fix es2015 imports

#1053 - moved here

### CODE REVIEW

File: packages/mdc-menu/README.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ mdc.MDCSimpleMenu.attachTo(document.querySelector('.mdc-simple-menu'));
 Simple menus can easily be initialized using their default constructors as well, similar to `attachTo`.
 
 ```javascript
-import MDCSimpleMenu from 'mdc-menu';
+import MDCSimpleMenu from '@material/mdc-menu';
```
Comment (kfranqueiro) [inline packages/mdc-menu/README.md]: Is this supposed to be `import {MDCSimpleMenu}`? I don't think our component modules have a default export.

Comment (Teoxoy) [inline packages/mdc-menu/README.md]: Oh yeah.

**Comment by Teoxoy** [comment]
Sure. I thought I fixed all of them but it seems i overlooked some.

**Comment by Teoxoy** [comment]
Np

**Comment by Teoxoy** [comment]
Won't that require to change all of the imports (es2015, commonJS and AMD) + other stuff in the READMEs ?

**Comment by kfranqueiro** [comment]
RE all 3 import types, that's a really good point.  It appears as though what you brought up is already reflected in a couple of READMEs e.g. [dialog](https://github.com/material-components/material-components-web/blob/b4e8b5a23efad7e4d6035c5cf325a5d8726102ef/packages/mdc-dialog/README.md#including-in-code).

RE "other stuff", what else in particular came to mind?

I think for the purposes of this PR we should at _least_ fix it for all of the ES2015 cases since that's what this was already focused on.  I think updating the other two cases in each README as well would be ideal, given that they're all probably located rather close to example code you'll have to update anyway, but let me know if you'd rather leave the AMD and CommonJS instances for a separate issue/PR.

Thanks, and apologies again for all the back-and-forth.

**Comment by Teoxoy** [comment]
Np, I can update all 3 import types and maybe the examples too. What I meant by "other stuff" were mainly examples.

File: packages/mdc-drawer/README.md [hunk 2]
```diff
@@ -335,7 +335,7 @@ correct drawer behaviors into idiomatic components.
 ##### ES2015
 
 ```javascript
-import {MDCTemporaryDrawer, MDCTemporaryDrawerFoundation, util} from 'mdc-drawer';
+import {MDCTemporaryDrawer, MDCTemporaryDrawerFoundation, util} from '@material/mdc-drawer';
```
Comment (kfranqueiro) [inline packages/mdc-drawer/README.md]: `@material/drawer` (remove `mdc-`)

File: packages/mdc-drawer/README.md [hunk 3]
```diff
@@ -379,7 +379,7 @@ mdc.drawer.MDCTemporaryDrawer.attachTo(document.querySelector('.mdc-temporary-dr
 Temporary drawers can easily be initialized using their default constructors as well, similar to `attachTo`.
 
 ```javascript
-import {MDCTemporaryDrawer} from 'mdc-drawer';
+import {MDCTemporaryDrawer} from '@material/mdc-drawer';
```
Comment (kfranqueiro) [inline packages/mdc-drawer/README.md]: `@material/drawer` (remove `mdc-`)

**Comment by Teoxoy** [comment]
I will fix the things I overlooked but should I leave the commonjs and amd imports as they are or revert the commits ?

**Comment by kfranqueiro** [comment]
Sorry, I realized shortly after posting that my wording could've been unclear.

I meant revert the AMD/CommonJS changes to have this PR only change ES2015 as originally planned, thanks.

**Comment by Teoxoy** [comment]
Ok np, I will do it in a few hours.

**Comment by kfranqueiro** [comment]
Thanks for all your patience during the review process on this. This looks good to me now; I merged master into the branch to ensure there were no conflicts, and I'll merge once the checks complete.

**Comment by Teoxoy** [comment]
Happy to help.

**Pull request merged**